cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -20,14 +22,24 @@
  * Cruise Control will use {@link HttpSession} to keep track the progress of such requests and
  * report the progress to the users.
  */
+@JsonResponseClass
 public class OperationProgress {
-  private static final String STEP = "step";
-  private static final String DESCRIPTION = "description";
-  private static final String TIME_IN_MS = "time-in-ms";
-  private static final String COMPLETION_PERCENTAGE = "completionPercentage";
+  @JsonResponseField
+  protected static final String OPERATION = "operation";
```

Comment:
question: Why do `OPERATION` and `OPERATION_PROGRESS` need to become protected variables?